### ISSUE TITLE
Feature request: Add an option to control the size of the plaques

### ISSUE THREAD

**Issue by EGOIST1372** [AHilyard/AdvancementPlaques#23]
self explanatory :D
Thanks in advance <3

**Comment by BLUEChester** [comment]
Yes yes yes! I literally came into the GitHub to request this exactly, would love to make it the size of the vanilla advancement but just with the fancy animation, this mod adds!

Thanks for your work on this awesome mod

**Comment by Puffadillo** [comment]
I also came here to request this feature. Would love to change the scale of the plaques to 0.7-0.8, as they're pretty big and obtrusive by default. A config option would be much appreciated if possible! :)

**Comment by AHilyard** [comment]
This issue has been moved to discord: https://discord.gg/UCKjnamDaW
I am much more active there, and it should get more attention there. Thank you for understanding.
